start_linux.sh

```diff
@@ -21,7 +21,9 @@ if !command -v python3 &> /dev/null; then
     exit
 fi
 
+pip3 install -r requirements.txt
```

Comment:
pip should be invoked via `python3` not using `pip3`. As `pip3` doesn't always point to the correct python install you want to use. Thus, to make sure you are installing the dependencies to the proper location do `python3 -m pip install -r requirements.txt`

Reply:
ok